js/templates/item.html

```diff
@@ -41,6 +41,11 @@ <h2 class="page-contractTitle noOverflow textOpacity1 marginLeft20 marginBottom8
             <%= polyglot.t('BUYNOW') %>
           </div>
         </li>
+        <% if (ob.totalReviews) { %>
+        <li class="border0 alignCenter padding10 marginBottom12 js-itemRating">
+          <span class="marginRight3"><%= ob.totalReviews %> Review<% ob.totalReviews !== 1 && print('s') %></span><% print(ob.starsTemplate({ rating: ob.avgRating })) %><span class="marginLeft2">(<%= ob.avgRating %>/5)</span>
```

Comment:
gosh, then maybe we should have review && reviewPlural in the lang file? '1 Reviews' would look really odd. I suppose we could also restructure the phrase, maybe 'Reviews: X'?